lib/easy_ai.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class EasyAiPlayer
+  attr_reader :name, :marker
+
+  def initialize(board, player_marker)
+    @name = 'Computer'
+    @marker = 'X'
+    @player_marker = player_marker
```

Comment:
Why do you need the `player_marker` here?

Reply:
Originally I was going to use `player_marker` to find the first available space that contained neither marker but this is no longer necessary -- I've updated `EasyAI`'s `select` method so that it calls a method in the `Board` class to check for a free space